src/utils/utils.js

```diff
@@ -24,6 +24,9 @@ export const CTGROUPS = {
     BR: CTYPES.BR,
 };
 
+export const UrlMatch = /((?:https?:\/\/)?(?:[a-z0-9-]{1,63}\.){1,2}[a-z]{2,15}(?:\/[^\s]*)?)/gi;
+export const UrlMatchWithInfos = /((https?:\/\/)?([a-z0-9-]{1,63}\.){1,2}[a-z]{2,15}(\/[^\s]*)?)/gi;
```

Comment:
1. I would call it "url regex" rather than "url match".
2. For constants, we use all caps style: `URL_REGEX` and `URL_REGEX_WITH_INFOS`.